apps/api/src/hyp3_api/validation.yml.j2

```diff
@@ -0,0 +1,7 @@
+{% for job_spec in job_types.values() %}
+{{ job_spec['api_job_type'] }}: {% if not job_spec['validators'] %}[]{% endif %}
```

Comment:
the "no validators" case reads a little weird, maybe leave the job type out entirely if it doesn't have validators, and use `dict.get()` in the API code?